moped-editor/package-lock.json

```diff
@@ -5857,29 +5857,22 @@
       }
     },
     "@material-ui/core": {
```

Comment:
Non-issue, more of a discussion reminder. 💭 

I'm not sure I 100% know the best practices about when and how to check in the package lockfile. I am not aware of anyone doing anything related to updating libraries, so I think this form of the lockfile will be fine to move into main. Maybe this is something we can touch on in an upcoming dev sync - that would be helpful to me.

Reply:
I checked in the package lockfile because I had to update the version of material ui, which means the lockfile changed -- this is a good discussion to have because I just figured if the package.json was changed, then the package-lock needed to also be included, but I don't know if thats best practice or what the alternative are.